evalml/tests/objective_tests/test_objectives.py

```diff
@@ -99,7 +99,7 @@ def test_can_get_only_core_and_all_objective_names():
 
```

Comment:
Gini coefficient is also being tested in `test_objective_outputs` so I think it's in line with how we test our other objectives.